frontend/cypress/integration/login_page_spec.js

```diff
@@ -0,0 +1,12 @@
+describe("The Login Page", function() {
+  it("Fails on bad login credentials", function() {
+    cy.visit("/"); // change URL to match your dev URL
+    cy.get('input[name="email"]').type("Nonexistent email");
+    cy.get('input[name="password"]').type("password");
+    cy.get('button[type="submit"]').click();
+
+    cy.wait(3000);
```

Comment:
Nerekomenduojama naudoti wait - kuo daugiau pauzių, tuo ilgiau testas runnins. Tik būtiniausiais atvejais šitą imam, kuriuo momentu lūžta testas, jei wait neįdedi?

Reply:
Yeah, cia visai crap, tik as nesugalvojau kaip gerai padaryti kad sulauktu kol baigs loadint ir netestuotu implementation, e.g. neiskotu kokio tago kuris turi klase `MuiFormHelpterError` or something. :D 
Reik daugiau docsu pasiskaityt. Bet jei tu zinai kaip gerai padaryt, tai feel free to fix

Reply:
Okay, pažiūrėsiu 👍